### PR TITLE
#185172187 A seller should be able to delete an item in his collection

### PR DESCRIPTION
**Pull request by bstwagir**
### What does this PR do?
This PR adds the delete product functionality
### Description of Task to be completed
A user, in this case, a seller will be able to delete a product they no longer want to see displayed in their collection. 
### How should this be manually tested?
* Navigate to the login page and sign in, if confirmed the user is a seller they have an option of viewing their collection.
* From the collection a single product can be accessed, via a 'delete button present on the page they will be able to delete the chosen product.
### What are the relevant pivotal tracker/Trello stories?
#185172187

### CODE REVIEW

**Review by ruberwa — APPROVED** [review]

**Review by ruberwa — CHANGES_REQUESTED** [review]
- [x] Functionalities are okay only a few small things to fix, but one thing which is affecting your task and can do the same to the entire work time it goes to develop, We are using SASS as a team why Tailwind? I think you can take care of it first!

**Review by ruberwa — DISMISSED** [review]

**Review by nshutip — APPROVED** [review]

File: src/redux/actions/proDetails.js
Comment (Raphaela664) [inline src/redux/actions/proDetails.js:1]: Change the baseUrl to the deployed one

**Review by Raphaela664 — CHANGES_REQUESTED** [review]

File: src/redux/actions/proDetails.js
Comment (bstwagir) [reply src/redux/actions/proDetails.js:1]: The deployed URLs have been reinstated.

**Review by Raphaela664 — APPROVED** [review]
Well done!

**Review by Raphaela664 — APPROVED** [review]

**Review by JulesNIYOMWUNGERI — APPROVED** [review]

**Pull request merged**